### ISSUE TITLE
"Playback in feeds" is missing from the General tab in settings

### ISSUE THREAD

**Issue by mishu-uu-ulala** [inotia00/ReVanced_Extended#278]
### Type

Error at runtime

### Application

ReVanced Extended v18.05.40
ReVanced Extended v18.06.35

### Bug description

Hi, I was using ReVanced Extended v18.3.something before I jumped into v18.05.40 yesterday. 
This v18.05.40 is the first time I'm faced with this problem. I can't stop the automatic video playback, when I'm in my home feed.
In previous versions, there was this option called "Playback in feeds" in the General section on Settings. 
![Screenshot_2023-02-19-13-37-18-862_com google android youtube](https://user-images.githubusercontent.com/101135662/219935564-da108049-dff0-45b1-ad52-23f38347bc10.jpg)

Now, for some reason It's gone.

Please help. Bring back Playback in feeds. It's really essential in my daily use.

### Steps to reproduce

Step 1. Tap your account avatar on the upper right corner in the screen.

Step 2. Tap "Settings"

Step 3. Tap the first option called "General"

Now try to look for "Playback in feeds". It's not there anymore.

### Relevant log output

```shell
ReVanced Extended v18.05.40
ReVanced Extended v18.06.35
```

### Screenshots or videos

![Screenshot_2023-02-19-13-37-18-862_com google android youtube](https://user-images.githubusercontent.com/101135662/219935892-c8895501-a00a-4686-8ff7-b8acaeae0410.jpg)

### Solution

Please bring back "Playback in feeds"

### Device Environment

Device            : Xiaomi Redmi Note 7 Pro
RAM/ROM    : 6/64 GB
OS                   : Android 10
Flavour           : MIUI 12.5.1 (Global-Stock)
Environment : Rooted

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by NBruderman** [comment]
Much more settings are missing, like notification, billing etc

**Comment by inotia00** [comment]
https://github.com/inotia00/ReVanced_Extended/issues/276

**Comment by mishu-uu-ulala** [comment]
> Much more settings are missing, like notification, billing etc

Through 6 to 7 version testing and several reboots, I think finally I somewhat pinpointed the problem. 

The missing options are consistent in magisk modules. I tested through v18.05.34 to v18.06.35 and in every single magisk module installation, ❝missing option❞ issue is consistent. 

On the other hand, I tested through v18.05.34 to v18.06.35 non-root apk files. These variants are perfectly fine. No ❝missing option❞ issues whatsoever.
